README.md

```diff
@@ -24,7 +24,7 @@ This repository contains the source code for the PyCon India 2025 website, built
 
 3. Start the development server:
    ```sh
-   npm run serve
+   make deploy-dev-server
```

Comment:
how is this change relevant to the PR?

Reply:
It's not related to this PR, but since it's a small change doc change I combined it along with this PR. Shall I push this changes as a different PR?